docs/explanations/faq.md

```diff
@@ -219,6 +219,11 @@ This is the canonical list of keyboard shortcuts:
 			<td><kbd>Esc</kbd></td>
 			<td><kbd>Esc</kbd></td>
 		</tr>
+		<tr>
+			<td>Select text between multiple blocks.</td>
+			<td></td>
+			<td><kbd>Shift</kbd>+<kbd>Arrow (⇦, ⇧, ⇨, ⇩)</kbd></td>
+		</tr>
```

Comment:
It might be good to move this up above the clear selection shortcut. This way the shortcuts related to selecting text are together and the clear selection shortcut still follows those.